server/i18n/en.json

```diff
@@ -6691,12 +6703,20 @@
     "translation": "Failed to fetch user report."
   },
   {
-    "id": "app.report.retrieve_batch_report.license_error",
-    "translation": "Batch reporting export only available to Pro and Enterprise."
+    "id": "app.report.send_report_to_user.export_finished",
+    "translation": "Your export is ready. {{.DataType}} for {{.DateRange}}. Click on the link below to download the report."
```

Comment:
Do we need to have the `{{.DataType}}` argument here? Constructing a sentence like this is dicey for translators, and while I think `for {{.DateRange}}` might be okay for them to deal with, `{{.DataType}}` doesn't give them any context for how to translate it.

Given the only possible values of `DataType` are the one for this report and the empty string which will produce incorrect output, should we just hardcode that? Alternatively, I think we could split this into three sentences so that we only have to swap out the second one when something changes, but that's more work than just hardcoding "This report contains user data" for now